Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -854,13 +854,13 @@
 
 void WebPageProxy::switchFromStaticFontRegistryToUserFontRegistry()
 {
-    process().send(Messages::WebProcess::SwitchFromStaticFontRegistryToUserFontRegistry(fontdMachExtensionHandle(SandboxExtension::MachBootstrapOptions::EnableMachBootstrap)), 0);
+    process().send(Messages::WebProcess::SwitchFromStaticFontRegistryToUserFontRegistry(fontdMachExtensionHandle(process(), SandboxExtension::MachBootstrapOptions::EnableMachBootstrap)), 0);
 }
 
-SandboxExtension::Handle WebPageProxy::fontdMachExtensionHandle(SandboxExtension::MachBootstrapOptions machBootstrapOptions)
+SandboxExtension::Handle WebPageProxy::fontdMachExtensionHandle(const WebProcessProxy& process, SandboxExtension::MachBootstrapOptions machBootstrapOptions)
```

Comment:
Could this function be on the WebProcessProxy instead of the WebPageProxy to avoid confusion?

Reply:
> Could this function be on the WebProcessProxy instead of the WebPageProxy to avoid confusion?

Yes, that's a really good point. I have fixed this in the latest patch.

Thanks for reviewing!